examples/migrating_trench/sediment_callback.py

```diff
@@ -0,0 +1,165 @@
+from thetis.callback import DiagnosticCallback
```

Comment:
I think we discussed moving this to the test rather than the example...

Reply:
I have moved the callback to the test